README.md

```diff
@@ -74,6 +74,7 @@ Builder function must accepts one argument with `ComponentDiagram` instance and
 - `.highlight(nodeId | [node1, node2, ...])` - highlights node(s) with provided `nodeId` and inbound/outbound arrows
 - `.clearHighlights()` - clears node highlightning
 - `.focus(nodeId)` - shows arrows relative to node with `nodeId` and hides others
+- `.scrollTo(nodeId | [node1, node2, ...])` - scroll the graph to node or set of nodes
```

Comment:
When `highlight`, `focus`, `expand`, or `makeRoot` is called, `scrollTo(nodeId)` should be applied automatically. Don't you agree?

Reply:
Sounds reasonable